regression/contracts/loop_contracts_do_while/test.desc

```diff
@@ -7,3 +7,4 @@ main.c
 --
 --
 This test checks that loop contracts work correctly on do/while loops.
+Fails because contracts are not yet supported on do while loops.
```

Comment:
Contracts are supported on do-while thanks to latch normalization, here you need to fix the loop invariant to `0 <= x && x < 10`

Reply:
Fixed for the dfcc variant.